documentation/api/ext/v1/resource-graphs.markdown

```diff
@@ -0,0 +1,151 @@
+---
+title: "PuppetDB 3.2 » Extensions API (PE only)"
+layout: default
+canonical: "/puppetdb/latest/api/ext/v1/resource-graphs.html"
+---
+
+[paging]: ../../query/v4/paging.html
+[query]: ../../query/v4/query.html
+[subqueries]: ../../query/v4/operators.html#subquery-operators
+[environments]: ../../query/v4/environments.html
+[nodes]: ../../query/v4/nodes.html
+[statuses]: /puppet/latest/reference/format_report.html#puppettransactionreport
+
+You can query resource-graphs by making an HTTP request to the
+`/pdb/ext/v1/resource-graphs` endpoint.
+
+## `/pdb/ext/v1/resource-graphs`
+
+This will return a JSON array containing all the resource-graphs for each node
+in your infrastructure.
+
+This resource-graph is a report and a catalog munged together so that all
```

Comment:
How about "This resource-graph is a unified view of data from the report and the catalog..."